### DISCUSSION TITLE
Can I Install Artisan without Admin Privileges?

### DISCUSSION THREAD

**Discussion by nathanjsolomon** [artisan-roaster-scope/artisan#638] [Q&A]
I am just one of several users on my computer (Windows 10) and don't have Admin Privliges on my computer and was wondering if there was a way to have all the requisite files in a folder to downloads to be able to run artisan without admin rights.

**Comment by MAKOMO** [comment]
Dear ??

> On 15. Jul 2021, at 15:24, nathanjsolomon ***@***.***> wrote:
> 
> 
> I am just one of several users on my computer and don't have Admin Privliges on my computer and was wondering if there was a way to have all the requisite files in a folder to downloads to be able to run artisan without admin rights.

This depends on the platform you are working on, one of the informations you forgot to disclose. On macOS you can install Artisan by a simple drag and drop to any directory. No Admin rights needed. However, this might not be much useful as to access any external device via a network or serial port interface you might need Admin rights (depending on how loose or tight your system is configured) to allow Artisan the corresponding access.

M

**Reply by nathanjsolomon** [reply]
Oh yeah sorry. I'm on Windows 10, that would have been a good bit of information in include! Sorry.

**Reply by poundy** [reply]
Then the answer is no, Artisan on Windows is installed via an installer and needs rights to do so. If all you're trying to do is have regular user(s) not be able to install or update it, but some admin can do that, then it works fine that way.... but you personally, not having admin rights, are not going to succeed installing it.
